sympy/stats/frv_types.py

```diff
@@ -785,7 +785,7 @@ def set(self):
 
     @property
     def is_symbolic(self):
-        return not all([self.k.is_number, self.c.is_number, self.delta.is_number])
+        return not all((self.k.is_number, self.c.is_number, self.delta.is_number))
```

Comment:
```python
return not (self.k.is_number and self.c.is_number and self.delta.is_number)
```
or perhaps
```python
return not all(s.is_number for s in (self.k, self.c, self.delta))
```
Not sure if classes in stats have something like `self.parameters` for this.